runtime/calamari/src/weights/xcm/mod.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2020-2022 Manta Network.
```

Comment:
Is this file automatically generated as well?

Reply:
No, this one is manual.